internal/pack_neon.h

```diff
@@ -243,22 +243,27 @@ class PackingRegisterBlock<
   }
 };
 
-// The format below is only used by one ARM64 kernel.
-// It uses a NEON intrinsic that is only available on ARM64, vpaddq_s16.
-// Would be easy to replace with vpadd_s16 if needed, would be almost as
-// efficient.
-#ifdef GEMMLOWP_NEON_64
```

Comment:
I don't understand the code in any depth, but was the removal of the 64-bit conditional and replacement with a 32-bit code block deliberate? I'm just a bit surprised to see a 64-bit chunk of code removed in a change I thought was only adding support for 32-bit.